playbooks/set-selinux-labels.yml

```diff
@@ -4,6 +4,12 @@
   gather_facts: no
 
   tasks:
+  - name: Deleting existing SELinux label if any
+    shell: semanage fcontext -d {{ item.path }}
+    with_items: "{{ mntpath }}"
+    when: mntpath is defined
+    ignore_errors: yes
```

Comment:
ingnoring error? If delete selinux lebel has some issue, which means this task will not mark as failure as error is ignored, then next task will fail right while applying same lebel again?
Why can't simply check if the paricular label is already set then ignore?

Reply:
@gobindadas the delete selinux label throws error only if that mount point is not labeled,
(the intial case) i.e when mountpath is not labeled in that case the next task will not fail.

Only that dependence is there, which makes the second task safe.
thats why i implemented it.